src/domain/entities/note.ts

```diff
@@ -43,6 +44,10 @@ export interface Note {
    * Last time note was updated
    */
   updatedAt: string;
+  /**
+   * Note settings
+   */
+  noteSettings?: NoteSettings | null;
```

Comment:
why note setting became part of the Note entity?

Reply:
Because, it was needed to extract noteSettings from note, while transforming note to notePublic and send it to client

Reply:
`settings = {
      customHostname: note.noteSettings.customHostname,
      isPublic: note.noteSettings.isPublic
    };`